### DISCUSSION TITLE
essay/2023/11/01/red-ocean

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#143] [Comments]
# essay/2023/11/01/red-ocean

앱 시장은 정말 레드오션일까? 그런 말을 들을 때마다 생각에 잠깁니다.

https://jeho.page/essay/2023/11/01/red-ocean.html

**Comment by juhwanlee02** [comment]
재밌는 글귀 감사합니다.
